wcfsetup/install/files/lib/util/HeaderUtil.class.php

```diff
@@ -47,8 +47,15 @@ public static function setCookie($name, $value = '', $expire = 0) {
 		if ($addDomain && strpos($cookieDomain, ':') !== false) {
 			$cookieDomain = explode(':', $cookieDomain, 2)[0];
 		}
+	    
+	    	$sameSitePolicy = "";
+	    	$sameSiteOption = COOKIE_SAME_SITE_ATTRIBUTE;
+	    	if ($sameSiteOption != "none") {
```

Comment:
I wonder if we forcibly should set some cookies to `SameSite` (notably the `cookieHash` cookie).

Reply:
Would be pleased to add it, if there's consent about this.